src/Request.php

```diff
@@ -2,40 +2,76 @@
 
 namespace Zumba\VanillaJsConnect;
 
-class Request {
-		protected $client_id;
-
-		public function __construct(array $args) {
-			if(isset($args['client_id'])) {
-				$this->client_id = $args['client_id'];
-			}
-
-			if(isset($args['timestamp'])) {
-				$this->timestamp = $args['timestamp'];
-			}
-
-			if(isset($args['signature'])) {
-				$this->signature = $args['signature'];
-			}
-
-			if(isset($args['callback'])) {
-				$this->callback = $args['callback'];
-			}
-		}
-		
-		public function getClientID() {
-			return $this->client_id;
-		}
-
-		public function getTimestamp() {
-			return $this->timestamp;
-		}
-
-		public function getSignature() {
-			return $this->signature;
-		}
-
-		public function getCallback() {
-			return $this->callback;
-		}
+class Request
+{
+    /**
+     * Holds the client ID
+     *
+     * @var string
+     */
+    protected $clientID;
+
+    /**
+     * Saves items from args array to object
+     *
+     * @param array $args
+     */
+    public function __construct(array $args)
+    {
+        if (isset($args['client_id'])) {
+            $this->clientID = $args['client_id'];
+        }
+
+        if (isset($args['timestamp'])) {
+            $this->timestamp = $args['timestamp'];
+        }
+
+        if (isset($args['signature'])) {
+            $this->signature = $args['signature'];
+        }
+
+        if (isset($args['callback'])) {
+            $this->callback = $args['callback'];
+        }
+    }
+
+    /**
+     * Gets client ID
+     *
+     * @return string
+     */
+    public function getClientID()
+    {
+        return $this->clientID;
+    }
+
+    /**
+     * Returns timestamp on Request
+     *
+     * @return integer
+     */
+    public function getTimestamp()
+    {
+        return $this->timestamp;
+    }
+
+    /**
+         * Returns Request signature
+         *
+         * @return string
+         */
```

Comment:
@j-feng13 There are several of these doc-blocks that look a little funky.  Can you straighten them?